lib/resources/mysql_session.rb

```diff
@@ -44,9 +44,9 @@ def query(q, db = '')
       cmd = inspec.command(command)
       out = cmd.stdout + "\n" + cmd.stderr
       if out =~ /Can't connect to .* MySQL server/ or
-         out.downcase =~ /^error/
+        out.downcase =~ /^error/
```

Comment:
That space is needed to appease Rubocop since it's a multiline conditional. We should also take this opportunity to change the `or` to `||` in the line above it.